Rock.Encryption/Symmetric/ProtocolExtensions.cs

```diff
@@ -36,5 +36,22 @@ public static byte[] ReadIVFromCipherTextHeader(this Stream stream)
             stream.Read(iv, 0, ivSize);
             return iv;
         }
+
+        public static bool IsEncrypted(this byte[] cipherText)
+        {
+            if (cipherText.Length < 3 || cipherText[0] != 1)
+                return false;
+
+            var ivSize = (ushort)(cipherText[1] | (cipherText[2] << 8));
+
```

Comment:
Seems like the value of ivSize can be between 0-2^16 (64K).  
(cipherText[2] << 8) is the high order bytes, and cipherText[1] is the low order bytes( 0-255).